docs/inquiries/overview.md

```diff
@@ -0,0 +1,6 @@
+Overview
+--------
+
+There are multiple ways you can work with us depending on your needs we can
+drive community edition firmware releases, organize sponsorware project or
+simply develop precisely what you need in scope of commercial project.
```

Comment:
simply develop precisely what you need in **the** scope of **a** commercial project.